marlowe-playground-client/webpack.config.js

```diff
@@ -107,7 +107,7 @@ module.exports = {
 
     resolve: {
         modules: [
-            'node_modules'
+            'node_modules', path.resolve(__dirname, './node_modules')
```

Comment:
Just curious why this is necessary? Is it to do with the symlink?

Reply:
Maybe i should add a comment about this in the code:

This adds the absolute path of `node_modules` in the respective client directory (plutus-playground or marlowe etc) to the search path for modules. Without it, node will fail to find some npm dependency that it tries to load from within the `web-common` directory.

Reply:
I added comments in https://github.com/input-output-hk/plutus/pull/2608/commits/977a20241e240bc3fe0cd6a2c0e943a4d78a095a